docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -259,8 +259,11 @@ For more details, read
 [Accessing Calico policy with Calico as a network plugin](AdvancedPolicy.md).
 
 ## Make a container reachable from the Host-Interface (Internet)
-
-You cannot simply use `-p`on `docker run` to expose ports. We have a working example on how to [expose a container port to the internet](../../ExposePortsToInternet.md)
+Since Calico creates new endpoints on the containers and does not use the `docker0`
+bridge, calling `docker run` with the `-p` flag will not function as expected.
```

Comment:
Not entirely true.  If you use default Docker IPAM, the Calico libnetwork plugin does connect tot he docker0 bridge.  Maybe we need to add this differentiator above in the select your ipam/network options